assets/schema_input.json

```diff
@@ -17,8 +17,8 @@
             "mlst_alleles": {
                 "type": "string",
                 "format": "file-path",
-                "pattern": "^\\S+\\.mlst\\.json(\\.gz)?$",
-                "errorMessage": "MLST JSON file from locidex report, cannot contain spaces and must have the extension: '.mlst.json' or '.mlst.json.gz'"
+                "pattern": "^\\S+\\.mlst(\\.subtyping)?\\.json(\\.gz)?$",
```

Comment:
Input files can either be `.mlst.json.gz` or `.mlst.subtyping.json.gz` as mikrokondo adds the `subtyping` string to the file name.